--- MODULE.bazel
+++ MODULE.bazel
@@ -22,8 +22,8 @@
 # Needed for bazelci and for building distribution tarballs.
 # If using an unreleased version of bazel_skylib via git_override, apply
 # MODULE.bazel-remove-override.patch to remove the following lines:
-bazel_dep(name = "bazel_skylib_gazelle_plugin", dev_dependency = True)
-local_path_override(
-    module_name = "bazel_skylib_gazelle_plugin",
-    path = "gazelle",
-)
+# bazel_dep(name = "bazel_skylib_gazelle_plugin", dev_dependency = True)
+# local_path_override(
+#     module_name = "bazel_skylib_gazelle_plugin",
+#     path = "gazelle",
+# )
